app-text/atril/atril-1.26.2.ebuild

```diff
@@ -0,0 +1,116 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+MATE_LA_PUNT="yes"
+
+PYTHON_COMPAT=( python3_{10..11} )
+
+inherit mate python-any-r1 virtualx
+
+if [[ ${PV} != 9999 ]]; then
+	KEYWORDS="~amd64 ~arm ~arm64 ~loong ~riscv ~x86"
+fi
+
+DESCRIPTION="Atril document viewer for MATE"
+LICENSE="FDL-1.1+ GPL-2+ GPL-3+ LGPL-2+ LGPL-2.1+"
+SLOT="0"
+
+IUSE="caja dbus debug djvu dvi epub +introspection keyring nls +postscript synctex t1lib test tiff xps"
+
+REQUIRED_USE="t1lib? ( dvi )"
+
+COMMON_DEPEND="
+	>=app-accessibility/at-spi2-core-2.46.0:2
+	>=app-text/poppler-0.22[cairo]
+	>=dev-libs/glib-2.62:2
+	>=dev-libs/libxml2-2.5:2
+	sys-libs/zlib
+	x11-libs/gdk-pixbuf:2
+	>=x11-libs/gtk+-3.22:3[introspection?]
+	x11-libs/libICE
+	>=x11-libs/libSM-1:0
+	x11-libs/libX11
+	>=x11-libs/cairo-1.14
+	x11-libs/pango
+	caja? ( >=mate-base/caja-1.17.1[introspection?] )
+	djvu? ( >=app-text/djvu-3.5.17:0 )
+	dvi? (
+		virtual/tex-base
+		t1lib? ( >=media-libs/t1lib-5:5 )
+	)
+	epub? (
+		<dev-libs/mathjax-3
+		>=net-libs/webkit-gtk-2.6.0:4.1
+	)
+	keyring? ( >=app-crypt/libsecret-0.5 )
+	introspection? ( >=dev-libs/gobject-introspection-0.6:= )
+	postscript? ( >=app-text/libspectre-0.2 )
+	synctex? ( virtual/tex-base )
+	tiff? ( >=media-libs/tiff-3.6:= )
+	xps? ( >=app-text/libgxps-0.2.1 )
+"
```

Comment:
You're missing `app-arch/libarchive:=`, per 
```
+COMICS_DEPS="libarchive"
 if test "x$enable_comics" = "xyes"; then
     AC_DEFINE([ENABLE_COMICS], [1], [Enable support for comics.])
+    PKG_CHECK_MODULES([COMICS], [$COMICS_DEPS])
+    AC_SUBST(COMICS_LIBS)
```

Reply:
Should I add a new flag `comics`?

Reply:
Mmmm that or just add `app-arch/libarchive:=` dep unconditionally. I presume comics is always enabled, is there a reason to make it toggleable? You decide.